modules/virt-how-virtual-gpus-assigned-nodes.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-mediated-devices.adoc
+
+:_content-type: CONCEPT
+[id="how-vgpus-are-assigned-to-nodes_{context}"]
+= How vGPUs are assigned to nodes
+
+For each physical device, {VirtProductName} configures:
```

Comment:
My suggestion might be the wrong phrase, but the sentence needs to be completed somehow :)

```suggestion
For each physical device, {VirtProductName} configures the following values:
```